Prow testing

humbledshuttler opened the pull request:
testing prow pre-submit

iamsubratp commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/iamsubratp/osconfig/pull/2#" title="Author self-approved">iamsubratp</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=iamsubratp%2Fosconfig).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/iamsubratp/osconfig/blob/master/OWNERS)~~ [iamsubratp]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

hopkiw committed:
no upgrade in deb build (#64)

TrevorBoone committed:
switching default permission constant to be specified correctly in octal format (#65)

manjunathganga committed:
Add fallback logic for installing/removing apt packages when applying… (#59)

* Add fallback logic for installing/removing apt packages when applying guest policy

* Move installing individual packages logic to the policies go package

TrevorBoone committed:
adding file extensions (and other fixes) to RunScript step (#70)

* switching default permission constant to be specified correctly in octal format

* adding extensions and fixing issue with arguments

* removing unnecessary comments

humbledshuttler committed:
Add tests for software recipes (#63)

* Add tests for software recipes

- Refactoring in software recipes to make code more testable
- Fixed a minor bugs
- Added unit tests for the package

* Fix merge issues

* Fix linter

* Remove function closures

* Fix formatter

* Implement feedback from reviewer

* Move common function to appropriate place

* Fix linter

* Implement feedback from reviewers

humbledshuttler committed:
Add more Unit tests (#60)

* Add tests for attributes and config package

* Improve coverage for unit test

* Add unit test for config, attributes, tasker package
* Fix bug in tasker:
  The tasker would never stop because the underlying channel
  was never closed and this the tasker waited on the channel
  forever

* Fix formatter

* Implement feedback from reviewer

* Implement feedback from reviewers

* Implement feedback from reviewers

hopkiw committed:
MSI support, update step detection (#66)

* MSI support, update step detection

* PR feedback

* make recipe functions private

hopkiw committed:
fix recipe initialization (#67)

adjackura committed:
Fix inventory error handling (#71)

* Fix inventory error handling

* fix windows build

Commit:
Test pr events for prow
